utils/validators.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable consistent-return */
```

Comment:
how come we need this now? we previously had functions in here that didn't have a `return` (i.e. only threw errors) and this wasn't an issue before

Reply:
Every other function except `validateRole` had a `return` at the end of the function. Furthermore, these functions did not have any if statements wrapping the return. In our case, we don't really care what the function returns for `validateLocationPins` and we have wrapped it in an if statement, which the linter doesn't like. `validateRole` is the only function that doesn't return anything and hence the linter doesn't complain. You can check out more details [here](https://eslint.org/docs/rules/consistent-return). 

I don't see a huge problem anyways because we are never using our function returns, unless I am missing something. Or, we can just return a boolean because it doesn't impact anything.